jenkins/e2e-image/e2e-runner.sh

```diff
@@ -462,11 +463,22 @@ if [[ "${FAIL_ON_GCP_RESOURCE_LEAK:-true}" == "true" ]]; then
   esac
 fi
 
+
 if [[ "${FEDERATION:-}" == "true" ]]; then
   FEDERATION_UP="${FEDERATION_UP:-true}"
   FEDERATION_DOWN="${FEDERATION_DOWN:-true}"
 fi
 
+
+# If we are only deploying a federation control plane without the federated
+# clusters, then we assume that the federated clusters are already deployed
+# and their kubeconfig is stored in GCS. We copy that kubeconfig from GCS
+# to the local machine to operate on those clusters.
+if [[ "${FEDERATION_UP:-}" == "true" && -z "${FEDERATION_CLUSTERS:-}" ]]; then
+  gsutil cp "${JENKINS_FEDERATION_PREFIX}/kube-config" "${HOME}/.kube/config"
+fi
+
```

Comment:
assumption about kube-config stored in GCS may be wrong. i could not find where kube-config is stored to GCS.
I only find in case of JENKINS_SOAK_MODE at line [536](https://github.com/kubernetes/test-infra/pull/1982/files#diff-6276ace2db72f1df8d04fb42c6989b69R536).